gtdynamics/dynamics/Chain.cpp

```diff
@@ -85,4 +85,80 @@ Pose3 Chain::poe(const Vector &q, boost::optional<Pose3 &> fTe,
   }
   return poe;
 }
+
+gtsam::Vector3 Chain::DynamicalEquality3(const gtsam::Vector6 &wrench,
```

Comment:
Biggest issue: these derivatives have not been unit-tested, so by default I do not believe them :-/
Check *many* examples of using numericalDerivative to check them.

Reply:
Jacobian updated and quite a lot of tests added as well